src/Classes/GemSelectControl.lua

```diff
@@ -209,15 +211,15 @@ function GemSelectClass:UpdateSortCache()
 				gemInstance.level = gemData.defaultLevel
 			end
 			--Calculate the impact of using this gem
-			local output = calcFunc({}, { allocNodes = true, requirementsItems = true, requirementsGems = true })
+			local output = calcFunc({}, { allocNodes = true, requirementsItems = true })
 			if oldGem then
 				gemInstance.gemData = oldGem.gemData
 				gemInstance.level = oldGem.level
 			else
 				gemList[self.index] = nil
 			end
 			-- Check for nil because some fields may not be populated, default to 0
-			sortCache.dps[gemId] = (output.Minion and output.Minion.CombinedDPS) or (output[dpsField] ~= nil and output[dpsField]) or 0
+			sortCache.dps[gemId] = (dpsField == "FullDPS" and output[dpsField] ~= nil and output[dpsField]) or (output.Minion and output.Minion.CombinedDPS) or (output[dpsField] ~= nil and output[dpsField]) or 0
```

Comment:
Same question here about the redundant logic
```suggestion
			sortCache.dps[gemId] = (output.Minion and output.Minion.CombinedDPS) or (output[dpsField] ~= nil and output[dpsField]) or 0
```

Reply:
Same answer as above, for niche situations where Mirages exist it wouldn't work. Since Mirages can have their own internal Minion tags. Example: General's Cry with Dominating Blow. The GC Mirage Warriors can apply the debuff triggering Sentinels.